src/days-2-4-adv/adv.py

```diff
@@ -49,3 +49,22 @@
 # Print an error message if the movement isn't allowed.
 #
 # If the user enters "q", quit the game.
+
+while True:
+    locked = 'No Entry'
+
+    print(f'{player.room.name}: {player.room.description}')
+
+    cmd = input('Which way would you like to go?')
+    if cmd == 'q':
+        break
+    elif cmd == 'n'
+        move north
+    elif cmd == 's'
+        move south
+    elif cmd == 'e'
+        move east
+    elif cmd == 'w'
+        move west
```

Comment:
Remember when you're doing this that the rooms are linked together, so if someone enters 'n', they should go north from whichever room they are in, but only if that is a valid option. Same for the other directions. Think of the current location and your n_to room links above.